modules/navier_stokes/src/auxkernels/kEpsilonViscosityAux.C

```diff
@@ -32,15 +32,17 @@ kEpsilonViscosityAux::validParams()
   params.addParam<Real>("C_mu", "Coupled turbulent kinetic energy closure.");
   params.addParam<std::vector<BoundaryName>>(
       "walls", {}, "Boundaries that correspond to solid walls.");
-  params.addParam<bool>(
-      "linearized_yplus",
-      false,
-      "Boolean to indicate if yplus must be estimate locally for the blending functions.");
   params.addParam<bool>("bulk_wall_treatment", false, "Activate bulk wall treatment.");
-  params.addParam<bool>(
-      "non_equilibrium_treatment",
-      false,
-      "Use non-equilibrium wall treatement (faster than standard wall treatement)");
+  MooseEnum wall_treatment("eq_newton eq_incremental eq_linearized neq", "eq_newton");
+  params.addParam<MooseEnum>("wall_treatment",
+                             wall_treatment,
+                             "The method used for computing the wall functions."
+                             "'eq_newton', 'eq_incremental', 'eq_linearized', 'neq'");
+  MooseEnum scale_limiter("none standard", "standard");
+  params.addParam<MooseEnum>("scale_limiter",
+                             scale_limiter,
+                             "The method used to limit the k-epsilon time scale."
+                             "'none', 'standard'");
```

Comment:
I thikn the `standard` limiting is not in the documentation. Please consider adding a more complete description here.

Reply:
Added to the documentation